src/server/data/automatedTestingData.js

```diff
@@ -234,14 +346,29 @@ async function generateTwentyThreeMinuteTestingData() {
 async function generateFifteenMinuteTestingData() {
 	const startDate = DEFAULT_OPTIONS.startDate;
 	const endDate = DEFAULT_OPTIONS.endDateOneYr;
+	const meterData = [
+		{
+			name: 'Sin 15 Min kWh',
+			unit: 'Electric_Utility',
+			defaultGraphicUnit: 'kWh',
+			displayable: true,
+			gps: undefined,
+			note: 'special meter',
+			// Store data in variable instead of filepath
+			data: [],
+			readingFrequency: '15 minutes',
+			area: 10,
+			areaUnit: 'feet',
+			deleteFile: true
+		},
+	]
 	const options = {
 		timeStep: { minute: 15 }, // Data point intervals set to 15 minutes.
 		periodLength: DEFAULT_OPTIONS.periodLength,
 		maxAmplitude: 3,
 		// Data saved in 'fifteenMinuteFreqTestData.csv' file.
-		filename: path.join(__dirname, '../test/db/data/automatedTests/fifteenMinuteFreqTestData.csv')
 	};
-	await generateSine(startDate, endDate, options);
+	insertData(startDate, endDate, options, meterData, conn);
 }
 
 /**
```

Comment:
While not part of the standard, generated data, it can be called via npm. The generated file is very large so I think we should see if it could be stored directly into the DB as with other functions (hopefully the variable is not too large).